compiler/src/dotty/tools/dotc/printing/PlainPrinter.scala

```diff
@@ -287,6 +287,8 @@ class PlainPrinter(_ctx: Context) extends Printer {
       case AnnotatedType(tpe, annot) =>
         if annot.symbol == defn.InlineParamAnnot || annot.symbol == defn.ErasedParamAnnot then toText(tpe)
         else toTextLocal(tpe) ~ " " ~ toText(annot)
+      case FlexibleType(tpe) =>
+        "FlexibleType(" ~ toText(tpe) ~ ")"
```

Comment:
Do we want some more concise notation? A principle is that flexible types should not be expressible in code, but maybe something more terse would be more readable. Kotlin uses `T?` but I'm not sure that's what we want for Scala.